### ISSUE TITLE
Transaction not submitted from Aiwa wallet pop up - shows amount as Token Symbol instead of AION

### ISSUE THREAD

**Issue by vaishnavipanchal23** [blockxlabs/aiwa#21]
#### AIWA version
1.2

#### Installed from Chrome Web Store or Github release?
web store

#### Chrome version? 
Version 70.0.3538.77 (Official Build) (64-bit)

#### OS and version?
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic

#### Which provider are you using?
Mastery

#### Expected vs actual behaviour

(GIF shown below) This transaction is a function interaction with the smart contract deployed.
Transaction not submitted from Aiwa wallet,
1. Shows unit in Amount field of popup as Token Symbol (MAVEN) of the contract instead of AION
2. Does not pick up Hex Data - even if it is set

![aiwa-issue](https://user-images.githubusercontent.com/30841481/53900637-ff626580-400a-11e9-98d3-e0ee90202823.gif)

#### Code snippets/screenshots
Contract Address: `0xa0ec2d8af8de8c4968641ed6597148cec80fa94e53cdac3f68fcbce0f05311a7`

```
if (!aiwaInjected) { // Check AIWA is enabled
      alert("You will need to have AIWA enabled to do the transfer");
    } else {
      // Create Tx object
      initiateTransferObj = {
        from: this.state.accounts,
        to: contractAddress,
        gas: 1000000,
        value: 0,
        data: myContract.methods.initiateFreezeTransfer(hash).encodeABI()
      }

      console.log('hex data:', myContract.methods.initiateFreezeTransfer(hash).encodeABI());
      // Prompt AIWA
      aiwa.eth.sendTransaction(
        initiateTransferObj
      ).then(function(txHash){
        console.log('txHash:', txHash);
        if (window.confirm('Click "OK" to see transaction hash.')) {
          window.open(
            'https://mastery.aion.network/#/transaction/'+ txHash,
            '_blank'
          );
        };
      });
```

#### Any additional information or details?
ABI of the smart contract shared offline with the team member!

**Comment by dhruvinparikh93** [comment]
#22

**Comment by vaishnavipanchal23** [comment]
Is there an update for this?

**Comment by kushptlapigarage** [comment]
Fixed in version 1.3, latest in Chrome Store.
